prototype/README

```diff
@@ -0,0 +1,7 @@
+Compiling and running the prototype (until we have a proper setup)
+
+1. Compile Dawn
+2. cd build && ./bin/unittest/DawnCUnittest --gtest_also_run_disabled_tests 2>../prototype/generated.hpp && clang-format -i ../prototype/generated.hpp
+   --> This will run the disabled CodeGenPlayground test that generated and outputs code
+3. cd prototype && g++ -o out driver.cpp grid.cpp -o out -std=c++17 -I <path/to/gtclang>/src; ./out
+4. paraview of_..vtk
```

Comment:
The very last frames are numerically broken... can't figure out why

Reply:
are they? I mean in this PR the examples doesn't do anything meaningful (just multiplying by 1.1 each timestep). In the icosahedral, the diffusion works in my case.

Reply:
ok, no problem then :)